src/course_bot/talk.clj

```diff
@@ -140,11 +140,12 @@
                            [[{:text "yes"} {:text "no"}]]}}))
 
 (defmacro if-parse-yes-or-no [tx token id text if-yes if-no]
-  `(cond
-     (= ~text "yes") ~if-yes
-     (= ~text "no") ~if-no
-     :else (do (talk/send-text ~token ~id "What (yes or no)?")
-               (talk/repeat-branch ~tx))))
+  (let [lower-text (str/lower-case text)]
+    `(cond
+       (= ~lower-text "yes") ~if-yes
+       (= ~lower-text "no") ~if-no
+       :else (do (talk/send-text ~token ~id "What (yes or no)?")
+                 (talk/repeat-branch ~tx)))))
```

Comment:
As CI prove it, it can't work because it is a macro. So text should be known at compile-time.